frontend/packages/console-app/src/status/snapshot.ts

```diff
@@ -1,7 +1,24 @@
-import { VolumeSnapshotKind } from '@console/internal/module/k8s';
+import { VolumeSnapshotKind, VolumeSnapshotContentKind } from '@console/internal/module/k8s';
 
-export const volumeSnapshotStatus = ({ status }: { status: VolumeSnapshotKind['status'] }) => {
+export const volumeSnapshotStatus = ({
+  status,
+}: {
+  status: VolumeSnapshotKind['status'] | VolumeSnapshotContentKind['status'];
```

Comment:
```suggestion
  status: VolumeSnapshotKind['status'];
```
They are the same. We could also refactor the status field from the type and make a common `VolumeSnapshotStatus` and use it in the two Kinds.